release/ray_release/scripts/ray_bisect.py

```diff
@@ -28,30 +28,37 @@ def main(test_name: str, passing_commit: str, failing_commit: str) -> None:
         )
         return
     commit_lists = _get_commit_lists(passing_commit, failing_commit)
-    blamed_commit = _bisect(test, commit_lists)
+    blamed_commit = _bisect(test, commit_lists, 3)
```

Comment:
define a var for the magic number 3? or add a comment on what 3 means here?

I guess it is concurrency?

if we always use 3, maybe just hard code it into `_biscet`?